src/lib/AnnotationControls.ts

```diff
@@ -145,10 +149,22 @@ export default class AnnotationControls {
         onClick({ activeControl: this.currentActiveControl, event });
     };
 
+    private handleKeyDown = (onReset: () => void) => (event: KeyboardEvent): void => {
+        if (event.key !== 'Escape' || this.currentActiveControl === AnnotationMode.NONE) {
+            return;
+        }
+
+        this.resetControls();
+        onReset();
+
+        event.preventDefault();
+        event.stopPropagation();
```

Comment:
This seems kind of risky on a document-level handler where other non-Preview listeners might live. What is the reasoning for it?

Reply:
Stop exiting preview modal.